scripts/release.py

```diff
@@ -10,7 +10,8 @@
 import ghutils
 from notifications import mail, mailgun, try_context
 
-logger = logging.getLogger("release")
+logging.basicConfig(level=logging.INFO)
```

Comment:
This appears to break the debug logging due to the ordering of `logging.basicConfig` calls between it and ghutils, I believe it only has an effect the first time it is called.